src/Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -176,14 +180,20 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
 
     public function configureOptions(OptionsResolver $resolver)
     {
-        $resolver->setDefaults([
+        $resolver
+            ->setDefaults([
             'sonata_admin' => null,
             'sonata_field_description' => null,
 
             // be compatible with mopa if not installed, avoid generating an exception for invalid option
             'label_render' => true,
+            // NEXT_MAJOR: Remove this property and the deprecation message
             'sonata_help' => null,
-        ]);
+        ])
+            ->setDeprecated(
+                'sonata_help',
+                'The option "sonata_help" is deprecated since sonata-project/admin-bundle 3.x, to be removed in 4.0. Use "help" instead.'
```

Comment:
```suggestion
                'The "sonata_help" option is deprecated since sonata-project/admin-bundle 3.x, to be removed in 4.0. Use "help" instead.'
```